com.unity.render-pipelines.universal/ShaderLibrary/DeclareNormalsTexture.hlsl

```diff
@@ -23,8 +23,8 @@ float3 LoadSceneNormals(uint2 uv)
     float3 normal = LOAD_TEXTURE2D_X(_CameraNormalsTexture, uv).xyz;
 
     #if defined(_GBUFFER_NORMALS_OCT)
-    half2 remappedOctNormalWS = Unpack888ToFloat2(normal); // values between [ 0,  1]
-    half2 octNormalWS = remappedOctNormalWS.xy * 2.0h - 1.0h;    // values between [-1, +1]
+    float2 remappedOctNormalWS = Unpack888ToFloat2(normal); // values between [ 0,  1]
```

Comment:
Since all of the code there is done in float I don't think I need to add a comment about half.
These lines were causing a precision loss warning in our shader so I removed the half variables.

Reply:
Ah yes, I missed the return type. It's all good then.